extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -165,9 +198,10 @@ export class CookieWriter {
    * Expand the value and write to cookie if necessary
    * @param {string} cookieName
    * @param {string} cookieValue
+   * @param {number=} opt_cookieExpireDateMs
    * @return {!Promise}
    */
-  expandAndWrite_(cookieName, cookieValue) {
+  expandAndWrite_(cookieName, cookieValue, opt_cookieExpireDateMs) {
```

Comment:
Looks to me that the input `cookieExpireDateMs` is always going to be a number. Do we still need to check for `opt_cookieExpireDateMs`?